roles/ansible-keylime-tpm20/tasks/git-repos.yml

```diff
@@ -0,0 +1,35 @@
+- name: Clone python-keylime
+  git:
+    repo: https://github.com/keylime/python-keylime
+    dest: /root/python-keylime
+
+- name: Clone tpm2 software stack
+  git:
+    repo: https://github.com/tpm2-software/tpm2-tss
+    dest: /root/tpm2-tss
+    version: a0d6f7d
```

Comment:
We should make this into a variable, see the following way of doing this:

https://github.com/lukehinds/ansible-tpm-simulator/blob/c25784ec8cbe4c2b55432d2bfc40a0b36929215f/roles/tpm2-simulator/vars/main.yml#L3

https://github.com/lukehinds/ansible-tpm-simulator/blob/c25784ec8cbe4c2b55432d2bfc40a0b36929215f/roles/tpm2-simulator/tasks/git-repos.yml#L6

It's also worth doing the same with the configure args:

https://github.com/lukehinds/ansible-tpm-simulator/blob/c25784ec8cbe4c2b55432d2bfc40a0b36929215f/roles/tpm2-simulator/vars/main.yml#L8-L10

https://github.com/lukehinds/ansible-tpm-simulator/blob/c25784ec8cbe4c2b55432d2bfc40a0b36929215f/roles/tpm2-simulator/tasks/tpm2-software-build.yml#L32

Reply:
Using vars makes more sense. Will change the args to vars.